src/components/VColorPaletteMaterial.vue

```diff
@@ -258,12 +269,7 @@ export default {
     height: 100% !important;
```

Comment:
Remove unnecessary css (replace with this): 
 .color-group-wrap {
  display: flex;
}
.color-group {
  display: flex;
  flex-direction: column;
}